config/cakephp.php

```diff
@@ -8,6 +8,13 @@
 
     'templatePath' => 'templates',
     'sourcePaths' => ['src', 'config'],
+    'excludes' => [
+        'namespaces' => [],
+        'names' => [
+            '\Cake\Collection\CollectionTrait',
+            '\Cake\I18n\DateFormatTrait',
```

Comment:
I would prefer to exclude more that are for internal developers such as EntityTrait and QueryTrait. Those would just confuse most users who are looking for Query or Entity.

Internal developers will be reading code.